examples/3d/atmosphere.rs

```diff
@@ -0,0 +1,110 @@
+//! This example showcases pbr atmospheric scattering
+
+use std::f32::consts::PI;
+
+use bevy::{
+    core_pipeline::tonemapping::Tonemapping,
+    pbr::{Atmosphere, AtmosphereSettings, CascadeShadowConfigBuilder},
+    prelude::*,
+};
+use light_consts::lux;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, (setup_camera_fog, setup_terrain_scene))
+        .add_systems(Update, dynamic_scene)
+        .run();
+}
+
+fn setup_camera_fog(mut commands: Commands) {
+    commands.spawn((
+        Camera3d::default(),
+        Camera {
+            hdr: true,
+            ..default()
+        },
+        Tonemapping::AcesFitted,
+        Transform::from_xyz(-1.2, 0.15, 0.0).looking_at(Vec3::Y * 0.1, Vec3::Y),
+        Atmosphere::EARTH,
+        AtmosphereSettings {
+            scene_units_to_km: 1.0,
+            ..Default::default()
+        },
+    ));
+}
+
+#[derive(Component)]
+struct Terrain;
+
+fn setup_terrain_scene(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+    asset_server: Res<AssetServer>,
+) {
+    // Configure a properly scaled cascade shadow map for this scene (defaults are too large, mesh units are in km)
+    let cascade_shadow_config = CascadeShadowConfigBuilder {
+        first_cascade_far_bound: 0.3,
+        maximum_distance: 3.0,
+        ..default()
+    }
+    .build();
+
+    // Sun
+    commands.spawn((
+        DirectionalLight {
+            color: Color::srgb(0.98, 0.95, 0.82),
+            shadows_enabled: true,
+            illuminance: lux::AMBIENT_DAYLIGHT,
+            ..default()
+        },
+        Transform::from_xyz(1.0, -1.0, 0.0).looking_at(Vec3::ZERO, Vec3::Y),
+        cascade_shadow_config,
+    ));
+
+    let sphere_mesh = meshes.add(Mesh::from(Sphere { radius: 1.0 }));
+
+    // light probe spheres
+    commands.spawn((
```

Comment:
The environment does not show up on the light probes, but this will be used as test subject in a future iteration where we generate the environment cube maps from sampling the sky-view LUT.